Lib/test/test_long.py

```diff
@@ -1693,5 +1693,11 @@ class MyInt(int):
         # GH-117195 -- This shouldn't crash
         object.__sizeof__(1)
 
+    def test_long_hash(self):
+        # gh-136599
+        assert hash(10) == 10
+        assert hash(-1) == -2
+        assert hash(-2**61) != -1
```

Comment:
```suggestion
        assertEqual(hash(10), 10)
        assertEqual(hash(-1), -2)
        assertNotEqual(hash(-2**61), -1)
```
Such methods are generally preferred (e.g. the above test) as they provide better errors.